crates/floresta-common/src/lib.rs

```diff
@@ -125,7 +137,10 @@ pub mod prelude {
     pub use hashbrown::HashMap;
     pub use hashbrown::HashSet;
 }
+
 #[cfg(feature = "std")]
+/// Provides commonly used items and traits in a centralized way,
+/// simplifying imports across the codebase.
```

Comment:
```suggestion
/// Provides implementation for basic `std` types, without assuming we have a `std` library.
///
/// This module is used to avoid having `#[cfg(feature = "no-std")]` sprinkled
/// around all crates that support `no-std`. It imports all types we would use
/// from the `stdlib`, either from the lib itself, or from other sources in case
/// `stdlib` isn't available.
```